test/fun.js

```diff
@@ -0,0 +1,16 @@
+const { describe, it, before } = require('mocha');
+const assert = require('assert');
+const { NewContext } = require('typescript-sdk/dist/mock/mock');
+const fetch = require('node-fetch');
```

Comment:
Please integrate TS-SDK for node-fetch, by adding the require stmt before this line.